[2024-10-02T21:15:08.574+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-02T21:15:09.893+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_preparation.read_data scheduled__2024-10-02T00:05:00+00:00 [queued]>
[2024-10-02T21:15:09.978+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_preparation.read_data scheduled__2024-10-02T00:05:00+00:00 [queued]>
[2024-10-02T21:15:09.982+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-02T21:15:10.245+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): read_data> on 2024-10-02 00:05:00+00:00
[2024-10-02T21:15:10.298+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=131) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T21:15:10.303+0000] {standard_task_runner.py:72} INFO - Started process 244 to run task
[2024-10-02T21:15:10.345+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_preparation', 'read_data', 'scheduled__2024-10-02T00:05:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpne0bq_sk']
[2024-10-02T21:15:10.383+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask read_data
[2024-10-02T21:15:11.389+0000] {task_command.py:467} INFO - Running <TaskInstance: data_preparation.read_data scheduled__2024-10-02T00:05:00+00:00 [running]> on host 1039cb6c9818
